.vscode/launch.test.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@ammaratef45 , could you please clarify what these changes are for? Do we need them in the scope of the issue #8?

Reply:
@denis-konovalyenko this file should be removed
I was using vscode and it added this file automatically and I didn't notice it

Reply:
@denis-konovalyenko I have removed it in a seperate commit